pxf/build.gradle

```diff
@@ -64,7 +64,7 @@ subprojects { subProject ->
             "-Xlint:finally", "-Xlint:overrides", "-Xlint:path", "-Xlint:processing", "-Xlint:static", "-Xlint:try",
             "-Xlint:-fallthrough", "-Xlint:-deprecation", "-Xlint:-unchecked", "-Xlint:-options"
     ]
-
+		
```

Comment:
Should we remove all whitespaces in the build.gradle file?  There are quite a few.